xbmc/settings/dialogs/GUIDialogSettingsManagerBase.cpp

```diff
@@ -18,12 +18,11 @@
  *
  */
 
-#include <set>
-#include <string>
-
 #include "GUIDialogSettingsManagerBase.h"
 #include "settings/lib/SettingsManager.h"
 
+#include <assert.h>
```

Comment:
`<cassert>`

Reply:
Is this general Kodi coding style? Because right now, the C++ header wrappers are rarely used. Again, I tried to replicate the coding style I observed in the current code.

Reply:
Again no hard rule, but you will note that https://codedocs.xyz/xbmc/xbmc/code_guidelines.html has `<cassert>` in the header explanation. As the C headers are deprecated in C++, it's my *personal opinion* that when refactoring anyway we should use the C++ wrappers.

Just for fun:

```
$ git grep cassert | wc -l
45
$ git grep assert.h | wc -l
30
```

I know that this makes learning the style from the current code very difficult, but I'm not sure what we can do about it short from refactoring everything (quite unrealistic) ...